platform-configs/include/protectli-vp32xx.robot

```diff
@@ -0,0 +1,15 @@
+*** Settings ***
+Resource    protectli-common.robot
+
+
+*** Variables ***
+${FLASH_SIZE}=                  ${16*1024*1024}
+${VERIFIED_BOOT_SUPPORT}=       ${TRUE}
+${IPXE_BOOT_ENTRY}=             Network Boot and Utilities
```

Comment:
@JanPrusinowski @SebastianCzapla It's already defined in protectli-common, right?

Reply:
You are correct. I have removed those entries in: e513d6b99380920e382407728ea9a30bb19a2a9f